db/fixtures/miq_product_features.yml

```diff
@@ -5123,6 +5123,54 @@
           :description: Edit Job Templates Tags
           :feature_type: control
           :identifier: configuration_script_tag
+
+# Embedded Automation Manager
+- :name: Embedded Automation Manager
+  :description: Everything under Ansible
+  :feature_type: node
+  :identifier: embedded_automation_manager
+  :children:
+  - :name: Repositories
+    :description: Everything under Repositorie
+    :feature_type: node
+    :identifier: configuration_script_source
```

Comment:
@Fryguy - The playbooks and repos are only for the embedded ansible - but I agree, that may change - so I added the 'embedded' prefix